R/install.R

```diff
@@ -87,6 +87,28 @@ repositories <-
     repositories
 }
 
+## is the docker container configured correctly?
+.test_container_bioc_versions <- function(version, container_version) {
+    bioconductor_version <- package_version(version)
+    docker_version <- package_version(container_version)
+    (bioconductor_version$major == docker_version$major) &
+        (bioconductor_version$minor == docker_version$minor)
+}
+
+## are we running on a docker container?
+.platform_bioc_docker_version <- function() {
```

Comment:
please name functions so that they reflect their role in the package `.repository_container_version()`?